package.json

```diff
@@ -135,7 +135,7 @@
 		"test:mocha": "NODE_ENV=test TEST_MODE=unit NEW_RELIC_ENABLED=false mocha",
 		"test:mocha:dev": "TS_NODE_TRANSPILE_ONLY=true NODE_ENV=test TEST_MODE=unit NEW_RELIC_ENABLED=false mocha",
 		"test:perf": "tsx test-perf/index.ts",
-		"test:portman": "TEST_DONT_RESTART_WORKERS=1 start-test 'npm run start:test' http://localhost:3000/health 'npm run test:portman:create && npm run test:portman:run' 2> /dev/null || E=$?; rm -rf tmp; exit $E;",
+		"test:portman": "TEST_DONT_RESTART_WORKERS=1 start-test 'npm run start:test' http://localhost:3000/health 'npm run test:portman:create && npm run test:portman:run' || E=$?; rm -rf tmp; exit $E;",
```

Comment:
I had an error during `test:portman` (can't remember which exactly), but the origins were hidden because of stderr redirect, so I want to make it explicit.